back/engines/commercial/id_nemlog_in/spec/requests/nemlog_in_verification_spec.rb

```diff
@@ -119,4 +120,30 @@
 
     expect(response).to redirect_to('/whatever-page?verification_error=true&error=no_token_passed')
   end
+
+  context 'when verification method is used by Copenhagen' do
```

Comment:
```suggestion
  context "when validating user's age" do
```
But it's up to you :smile: